bin/server.js

```diff
@@ -1,13 +1,14 @@
 import {ModuleLoader} from "./lib/ModuleLoader.js";
 import ApolloModule from "./modules/ApolloModule.js";
+import MailModule from "./modules/MailModule.js";
 import Table from 'cli-table3';
 import * as fs from 'fs';
 import colors from 'colors';
 import MongooseModule from "./modules/MongooseModule.js";
 
 colors.enabled = true;
 
-let vobeJson = JSON.parse(fs.readFileSync('vobe.json').toString());
+global.vobeJson = JSON.parse(fs.readFileSync('vobe.json').toString());
```

Comment:
Maybe we could move the vobeJson to an entire new file and export it from there.
So you only have to use
```js
import vobeJson from 'vobeJson.js'
```

Using global variables is fine at the moment, but can make things worse later.

Reply:
Do you mean that the variables are stored directly in the vobeJson.js files as follows?
```js
const variable1 = something;
const variable2 = something;
const variable3 = something;
```
Or do you mean that the vobeJson.js parses the vobe.json and saves it in a vobeJson variable like currently in server.js
```js
const vobeJson = JSON.parse(fs.readFileSync('vobe.json').toString());
```